requirements.txt

```diff
@@ -1,4 +1,4 @@
-tensorflow>=2.10.0
+tensorflow==2.10.0
```

Comment:
Any particular reason to fix 2.10?

Reply:
So that the training doesn't meet with unexpected surprises.